UWP/HockeyAppDemoUwp/cs/project.lock.json

```diff
@@ -11695,7 +11695,7 @@
       ]
     },
     "Microsoft.CSharp/4.0.0": {
-      "sha512": "/q7Wo1eEf1Nv2cuOO5h1F/MzEM0xn0MoV2kb1YpgDZFYZ/y5gkPYUA0Ws6mVADJKiMWnl2/RXb87roVhUy0QXA==",
```

Comment:
I don't think this file should be checked in. 
http://stackoverflow.com/questions/38065611/what-is-project-lock-json

Can you delete it, and filter it out in .gitignore?